pkg/services/librarypanels/models.go

```diff
@@ -22,11 +22,24 @@ type LibraryPanel struct {
 	UpdatedBy int64
 }
 
+// LibraryPanelDashboard is the model for library panel connections.
+type LibraryPanelDashboard struct {
```

Comment:
I think this type can be un-exported.
```suggestion
// libraryPanelDashboard is the model for library panel connections.
type libraryPanelDashboard struct {
```

Reply:
Fixed